internal/cache/metadata/type_cache.go

```diff
@@ -94,6 +94,14 @@ func (ce cacheEntry) Size() (size uint64) {
 	return
 }
 
+// Return the size (rss) of a type-cache entry
+// for a given key-string.
+func SizeOfTypeCacheEntry(key string) uint64 {
```

Comment:
rss?
shouldn't it be heap size?

Reply:
No, it's rss now. Look at the code of entry.Size() function.